test/ejabberd_tests/tests/privacy_SUITE.erl

```diff
@@ -711,6 +711,7 @@ block_jid_message_but_not_presence(Config) ->
         end).
 
 newly_blocked_presense_jid_by_new_list(Config) ->
+    reset_users(Config),
```

Comment:
Instead of reseting users you can use `escalus_fresh:story/3` instead of `escalus:story/3`. This will create new users for you based on the provided specs. Please remember to call `escalus_fresh:clean/0` in `end_per_suite` to remove this `fresh` users.